examples/example-provider-authentication/main.tf

```diff
@@ -1,11 +1,21 @@
 # ---------------------------------------------------------------------------------------------------------------------
-# PROVIDE CREDENTIALS TO GSUITE PROVIDER
+# PROVIDE CREDENTIALS TO PROVIDER
 # Credentials are stored in terraform.tfvars file.
 # ---------------------------------------------------------------------------------------------------------------------
 
-provider "gsuite" {
-  impersonated_user_email = var.impersonated_user_email
-  credentials             = var.credentials
+terraform {
+  required_providers {
+    googleworkspace = {
+      source  = "hashicorp/googleworkspace"
+      version = ""
+    }
+  }
+}
+
+provider "googleworkspace" {
```

Comment:
for me it looks like a bad practice to set the credentials in the repo vars as it increase the chance to commit them by accident. Also for ci systems it is harder to configure a temporary file then to use then ENV values.

There is an ENV configuration to read the credentials. with tools like [direnv](https://direnv.net/) it is easy to set it up for local development aswel as for ci systems.